teletest_api/app.py

```
@@ -0,0 +1,120 @@
+    for m in reversed(messages):
+        msgs.append(BotResponse(message_text=m.raw_text, reply_markup=_parse_buttons(m)))
+    return GetMessagesResponse(messages=msgs)
+
+
+@app.post("/reset-chat")
+async def reset_chat(req: ResetChatRequest) -> dict:
+    entity = await client.get_input_entity(req.bot_username)
+    try:
+        await client.delete_dialog(entity)
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Failed to reset chat: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)